github_pr.py

```diff
@@ -237,8 +255,11 @@ def main():
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if not args['numberonly'] and not args['noratelimit']:
-        print "Github Rate Limiting: %d remaining of max %d" % (gh.rate_limiting[0], gh.rate_limiting[1])
+    if (('numberonly' in args) and args['numberonly']):
```

Comment:
@cchristous what do you think of this? look better? i know we still have the "not args['noratelimit']" on line 261 but its a double negative since we are using the wording "noratelimit". this prints properly.